custom/abt/reports/late_pmt.py

```diff
@@ -139,66 +80,85 @@ def headers(self):
         )
 
     @cached_property
-    def smss_received(self):
-        data = SMS.objects.filter(
-            domain=self.domain,
-            couch_recipient_doc_type='CommCareUser',
-            direction=INCOMING,
-            couch_recipient__in=self.get_user_ids,
-            date__range=(
-                self.startdate,
-                self.enddate
-            )
-        ).exclude(
-            text="123"
-        ).values('date', 'couch_recipient').annotate(
-            number_of_sms=Count('couch_recipient')
-        )
-        return {(sms['date'].date(), sms['couch_recipient']) for sms in data}
-
-    @cached_property
-    def get_user_ids(self):
-        return [user['user_id'] for user in self.get_users]
-
-    @cached_property
-    def get_users(self):
-        return LatePMTUsers(config=self.report_config).get_data()
+    def pmts_submitted(self):
+        forms = list(iter_forms_by_xmlns_received_on(
+            self.domain, INDICATORS_FORM_XMLNS, self.startdate, self.enddate
+        ))
+        return {
+            (case.received_on, case.operation_site)
```

Comment:
Is `case.received_on` a thing?  I think this needs to be `form.received_on`, since subsequent updates to the same case would I assume overwrite `case.received_on`.

Reply:
Thank you for spotting that! 31848ce04d5

Reply:
Thank you for spotting that! 03155cebb33